src/components/chat/comment-card.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from "@testing-library/react";
+import { Provider } from "mobx-react";
+import React from "react";
+import { UserModelType } from "../../models/stores/user";
+import { ENavTab } from "../../models/view/nav-tabs";
+import { CommentCard } from "./comment-card";
+
+describe("CommentCard", () => {
+
+  const stores = { ui: { activeNavTab: ENavTab.kMyWork } };
+  const activeNavTab = stores.ui.activeNavTab;
```

Comment:
Should be able to remove the `stores` and all the `Provider` stuff now that it's being passed in as a prop.

Reply:
I think you can remove the two lines above as well.